docs/01-engage/02-accounts/02-creating-accounts/index.md

```diff
@@ -0,0 +1,113 @@
+
+- On the [ENGAGE dashboard](https://beta.n3o.cloud/eu1/engage/dashboard), navigate to **Quick Navigation**.
+
+![Create Account Quick Navigation](./create-account-quick-navigation.png)
+
+- Click on **Create Account** and a screen will appear to create the account.
+
+:::note 
+An account can be created for both individual and an organization. Each of them will have different fields to be covered. To follow along, make sure to have the correct information. 
+:::
+
+## Create an Individual Account
+
+:::note Important 
+- Input boxes with *asterisk* are compulsory to fill in. If you do not, an error message will appear e.g. *"State is a required field"*. 
+- By default, create new account screen always opens for an individual account.
+- Sometimes, while entering account information, background search might show previously entered accounts with a similar input.   
+:::
+
+1. In the *Name* section, specify the **Title**, **First Name** and **Last Name**.
+
+![Create Account Name](./create-account-1.png)
+
+2. Next, in the *Address* section, enter the **Country** you reside in, and it's respective **Postcode**. Input your address as **Line 1**, **Line 2** or **Line 3** and the **Town/City**.
+
+:::info
+- Most of the time, the address may appear relative to the postcode search.
+- For some countries, address input parameters might be different.
+:::
+
+![Create Account Address](./create-account-2.png)
+
+3. Moving forward, in the *Phone Number* section, enter your phone number along with the country code. Then select whether the number is a *fixed line* or *mobile*. You can also add extra phone numbers with **+ Add Phone Number** and remove them with the **Remove** button. 
+
+:::caution
+For incorrect and incomplete phone numbers, a **validation pop-up error message** will appear!
+:::
+
+![Create Account Phone Number](./create-account-3.png)
+
+4. In the *Email Address* section, input your *Personal* or *Work* email and select it respectively. You can also add extra email addresses with **+ Add Email Address** and remove them with the **Remove** button.
+
+![Create Account Email](./create-account-4.png)
+
+5. After this, in the *Online Presence* section, select the type from the drop-down including *Facebook*, *LinkedIn* and *Instagram* etc and state its example. You can add additional types using **Add Profile** and remove them as wanted with the **Remove** option.
+
+:::note 
+Online Presence refers to the combined existence of an individual or organization that is found online via search. 
+:::
+
+![Create Account Online Presence](./create-account-5.png)
+
+6. For the *Contact Preferences* section, select the way you want to receive communication i.e. via **Email**, **Post**, **SMS** or **Telephone**. Each category contains five different options named **Unspecified**, **Admin Only**, **Fundraising Only**, **All** and **None**. Select any category. For details, refer to the Contact Preferences(link to page) page.
```

Comment:
This isn't right. Those categories are not fixed. I think even for now this needs a subpage on contact preferences that you link to as need to explain not just the preference structure and presets, but also the consent bits that need explaining as confuse infrequent or new users.